### ISSUE TITLE
Unable to use pipewire/jack with libvirtd

### ISSUE THREAD

**Issue by Madouura** [NixOS/nixpkgs#149862]
### Describe the bug
Due to #114704 not being merged, and/or pipewire jack libs not overriding the libs in the qemu package.

### Steps To Reproduce
1. https://wiki.archlinux.org/title/PCI_passthrough_via_OVMF#Passing_VM_audio_to_host_via_JACK_and_PipeWire

### Additional context
I've tried compiling it myself in nixpkgs, however even though I can get jack working, am unable to get pipewire-jack working with qemu.
The reason I want to do this, is https://wiki.archlinux.org/title/PCI_passthrough_via_OVMF#Passing_VM_audio_to_host_via_JACK_and_PipeWire, which seems to require https://aur.archlinux.org/packages/pipewire-jack-dropin/.
I have not found a way to emulate that behavior with nix, unfortunately.

### Metadata
- system: `"x86_64-linux"`
- host os: `Linux 5.15.4-xanmod1-tt, NixOS, 21.11 (Porcupine)`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.3.16`
- channels(root): `"nixos-21.11.334139.1bd4bbd49be, home-manager-21.11, nixos-unstable-22.05pre335501.c71f061c68b"`
- channels(mado): `"home-manager-21.11, nixos-21.11.334027.e34c5379866, nixos-unstable-22.05pre335501.c71f061c68b"`
- nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by YellowOnion** [comment]
Try using LD_LIBRARY_PATH with:

```xml
<qemu:commandline>
   <qemu:env name='MY_VAR' value='my_value'/>
</qemu:commandline>
```

The AUR package just changes `/etc/ld.so.conf` so maybe possible to modify that with an override of some sort.

**Comment by WillPower3309** [comment]
The branch linked needs its conflicts to be fixed, then it should be good for review

**Comment by Madouura** [comment]
Theoretically fixed with #151233 merged and YellowOnion's suggestion, will test in a few days once I'm home.

**Comment by Madouura** [comment]
It works with ``<qemu:env name="LD_LIBRARY_PATH" value="$LD_LIBRARY_PATH:/run/current-system/sw/lib/pipewire"/>``!!!
Hallelujah!
